.github/workflows/release.yml

```diff
@@ -257,7 +257,7 @@ jobs:
 
   macos-test:
     name: 'Test MacOS Package'
-    runs-on: macos-13
+    runs-on: self-macos-13
```

Comment:
shouldn't these be `self-macos-12` ?

Reply:
![image](https://github.com/runtimeverification/k/assets/1329034/3c94d004-208b-4096-befd-41425787caa4)

We have an M2 runner on `macos-13`